components/src/Alert/__snapshots__/Alert.story.storyshot

```diff
@@ -2,3880 +2,237 @@
 
 exports[`Storyshots Alert Danger 1`] = `
 <div
-  style={
-    Object {
-      "padding": "24px",
-    }
-  }
+  style="padding:24px"
 >
-  <NDSProvider
```

Comment:
Changing from mount to render for the storyshots renderer removed all the theme data and the locale object from each storyshot